src/main/you-market-front/src/Producto-listado/styles.css

```diff
@@ -0,0 +1,63 @@
+.producto-list-container {
+    border: 1px solid gray;
+    padding: 20px;
+    max-width: 600px;
+    margin: 0 auto;
+    margin-bottom: 15px;
+    cursor: pointer;
+    box-shadow: 0 1px 3px #aaaaaa;
+}
+
+.producto-list-container::after {
+    content: "";
+    clear: both;
+    display: table;
+}
+
+.producto-list-imagen {
+    width: 150px;
+    height: 150px;
+    float: left;
```

Comment:
Si usáis la propiedad de css `object-fit: cover` se adaptará al tamaño que tenga disponible, evitando el efecto visual de estrechamiento.